# ObjectPose.msg

# Name of the object
string object

# Positions of the other objects relative to this object
qualification/KeyValue[] positions
